src/katello/enabled_report.py

```diff
@@ -14,22 +11,50 @@ def __generate(self):
         config = ConfigParser()
         config.read(self.repofile)
         enabled_sections = [section for section in config.sections() if config.getboolean(section, "enabled")]
-        enabled_repos = [{"repositoryid": section, "baseurl": [self._replace_vars(config.get(section, "baseurl"))]} for section in enabled_sections]
+        enabled_repos = [{"repositoryid": section, "baseurl": [self._format_str(config.get(section, "baseurl"))]} for section in enabled_sections]
         return {"enabled_repos": {"repos": enabled_repos}}
 
-    def __init__(self, repo_file=REPOSITORY_PATH):
+    def __init__(self, repo_file=REPOSITORY_PATH, tool=TOOL_YUM):
```

Comment:
This additional argument could be removed per my suggestion of YUM/ZYPPER booleans